src/main/webapp/app/exercises/shared/result/updating-result.component.ts

```diff
@@ -35,6 +35,8 @@ export class UpdatingResultComponent implements OnChanges, OnDestroy {
     @Input() showIcon = true;
     @Input() isInSidebarCard = false;
     @Input() showCompletion = true;
+    @Input() showProgressBar = false;
+    @Input() showProgressBarBorder = false;
```

Comment:
I think in general we should only add input signals for new code, but does not need to be changed in this PR

Reply:
I didn't want to add signals since the component wasn't migrated, and migrating the component would make this PR much bigger than it currently is